### DISCUSSION TITLE
Forcing an "Enabled" status for Global Set → Singles entrification?

### DISCUSSION THREAD

**Discussion by aaronbushnell** [craftcms/cms#12837] [Ideas] [answered]
We frequently reach for a Global Set when we want to allow a client to change things like the site favicon or a callout that shows up in the header/footer. 

As of Craft 4.4 the `php craft entrify/global-set` command will convert these to a **Single**. While that makes sense to me, I don't think it should be possible to set these Singles into a "Disabled" state. Instead, this type of material should always be "Enabled" like their Global Set counterpart was.

Unless there's a more intuitive AX experience I could be overlooking here!

**Comment by piotrpog** [comment]
Agreed, that makes sense.

**Comment by brandonkelly** [comment, marked as answer]
Craft 4.5.0 is out with a new “Show the Status field” setting on entry types. And it’s disabled by default for entry types that are created via `entrify/global-set`.

Note that Single entries still do technically have statuses when this is disabled; they are just not author-editable. You could change their status via the `resave/entries` command though.
